diff --git a/static/bedrock_viz.js b/static/bedrock_viz.js
index 5c145fa..b9cc376 100644
--- a/static/bedrock_viz.js
+++ b/static/bedrock_viz.js
@@ -2116,8 +2116,8 @@ function initDimension() {
             view: new ol.View({
                 projection: projection,
                 center: [
-                    dimensionInfo[globalDimensionId].playerPosX + Math.floor(dimensionInfo[globalDimensionId].worldWidth / 2)
-                   ,dimensionInfo[globalDimensionId].playerPosY + Math.floor(dimensionInfo[globalDimensionId].worldHeight / 2)
+                    dimensionInfo[globalDimensionId].playerPosX
+                   ,dimensionInfo[globalDimensionId].playerPosY
                 ],
                 resolution: 1
             })
@@ -2126,8 +2126,8 @@ function initDimension() {
         var view = new ol.View({
             projection: projection,
             center: [
-                dimensionInfo[globalDimensionId].playerPosX + Math.floor(dimensionInfo[globalDimensionId].worldWidth / 2)
-               ,dimensionInfo[globalDimensionId].playerPosY + Math.floor(dimensionInfo[globalDimensionId].worldHeight / 2)
+                dimensionInfo[globalDimensionId].playerPosX
+               ,dimensionInfo[globalDimensionId].playerPosY
             ],
         resolution: 1
         });
